test/NonSilo.Tests/Serialization/BuiltInSerializerTests.cs

```diff
@@ -415,9 +415,9 @@ public void Serialize_HashSetWithComparer(SerializerToUse serializerToUse)
             Assert.Equal(source1.Count, result.Count); //Count is wrong after round-trip of string hash set with comparer
             foreach (var key in source1)
             {
-                Assert.True(result.Contains(key)); //key is missing after round-trip of string hash set with comparer
+                Assert.Contains(key, result); //key is missing after round-trip of string hash set with comparer
             }
-            Assert.True(result.Contains("One")); //Comparer is wrong after round-trip of string hash set with comparer
+            Assert.Contains("One", result); //Comparer is wrong after round-trip of string hash set with comparer
```

Comment:
Build failure: Assert.Contains is probably enumerating and using an equality comparer, where as the original assertion was relying on the HashSet to use CaseInsensitiveStringEquality. Same with Serialize_SortedSetWithComparer. I would just revert all of the assertions that use Contains in this class.